internal/tracker/source/tekton/tekton.go

```diff
@@ -348,23 +351,40 @@ func PreparePackage(i *PreparePackageInput) (*hub.Package, error) {
 	var name, version, description, tektonKind string
 	var annotations map[string]string
 	var tasks []map[string]interface{}
+	var steps []v1beta1.Step
+	var containerImages []*hub.ContainerImage
+
 	switch m := i.Manifest.(type) {
 	case *v1beta1.Task:
 		tektonKind = "task"
 		name = m.Name
 		version = m.Labels[versionLabelTKey]
 		description = m.Spec.Description
 		annotations = m.Annotations
+
+		// Prepare container images
+		ts := m.TaskSpec()
+		var defaults []v1beta1.ParamSpec
+		if len(ts.Params) > 0 {
+			defaults = append(defaults, ts.Params...)
+		}
+		mts := taskrun.ApplyParameters(context.Background(), &ts, &v1beta1.TaskRun{}, defaults...)
+		steps = mts.Steps
 	case *v1beta1.Pipeline:
 		tektonKind = "pipeline"
 		name = m.Name
 		version = m.Labels[versionLabelTKey]
 		description = m.Spec.Description
 		annotations = m.Annotations
-		for _, task := range m.Spec.Tasks {
+
+		// Prepare container images
```

Comment:
Same, as we don't really prepare the container images here anymore.

Reply:
Updated!